constructor/build_outputs.py

```diff
@@ -30,7 +33,7 @@ def process_build_outputs(info):
                 f"Available keys: {tuple(OUTPUT_HANDLERS.keys())}"
             )
         outpath = handler(info, **config)
-        print(f"build_outputs: '{name}' created '{os.path.abspath(outpath)}'.")
+        logger.info(f"build_outputs: '{name}' created '{os.path.abspath(outpath)}'.")
```

Comment:
Do you think we should use %-style formatting for `logging` calls or stick to f-strings? I don't really know how the community is positioned in this aspect, but I think I recall %-format was preferred?

Reply:
Updated the PR description with some reasoning on this topic.